Xamarin.Forms.Platform.MacOS/Renderers/SliderRenderer.cs

```diff
@@ -212,5 +215,10 @@ void UpdateValue()
 			if (Math.Abs(Element.Value - Control.DoubleValue) > 0)
 				Control.DoubleValue = (float)Element.Value;
 		}
+
+		void UpdateRotation()
+		{
+			Control.BoundsRotation = (float)Element.Rotation;
```

Comment:
Could we review/move this transformation in **VisualElementTracker**?. `VisualElementTracker` manage all the view transformations (rotation, scale, etc.).

Reply:
Please note, this is my first PR.
I'll look into this.

Reply:
@JunielKatarn Of course, and we really appreciate your effort. For that reason, let me know if we can help you with something.

Reply:
@jsuarezruiz I tried using `VisualElementTracker` for this purpose, but ran into issues.
![image](https://user-images.githubusercontent.com/4507319/86910929-4ebd1800-c0cf-11ea-9f87-e1a3661bd9c6.png)

The problem is, the `BoundsRotation` property seems to be needed especially for `NSSlider`.
If I set that property to any NSView (within VisualElementTracker), rendering becomes incorrect as shown in the attached screenshot.

To get the right behavior, we want to set `BoundsRotation` only on the `NSSlider` control.
Thus, it would seem like the appropriate place for this is still the `SliderRenderer` class.

Note, I have not been able to achieve any rotation by setting the transform in VisualElementTracker for the macOS Slider. Again, `BoundsRotation` seems mandatory.